src/modules/cpu/kernel/non_silent_region_detection.hpp

```diff
@@ -0,0 +1,107 @@
+#include "rppdefs.h"
+
+Rpp32f getSquare(Rpp32f &value)
+{
+    Rpp32f res = value;
+    return (res * res);
+}
+
+Rpp32f getMax(std::vector<float> &values, int length)
+{
+    Rpp32f max = values[0];
+    for (int i = 1; i < length; i++)
+    {
+      max = std::max(max, values[i]);
```

Comment:
@Sampath please go over these tabbing issues carefully again. We've got 6 spaces here. Needs to be 8